### PR TITLE
Add react-geo to list of 3rd party libraries

### PR DESCRIPTION
**Pull request by marcjansen**
Title says it all. Please review.

### CODE REVIEW

**Comment by marcjansen** [comment]
Thanks for the review.

File: src/pages/3rd-party/index.hbs [hunk 1]
```diff
@@ -97,6 +97,11 @@ layout: main.hbs
             <td>A GWT wrapper for OpenLayers 3+ written in Java</td>
             <td><a href="https://github.com/TDesjardins">Tino Desjardins</a></td>
           </tr>
+          <tr>
+            <td><a href="https://github.com/terrestris/react-geo">react-geo</a></td>
+            <td>A set of geo related modules to use in combination with react, antd and OpenLayers</td>
```
Comment (elemoine) [inline src/pages/3rd-party/index.hbs]: Do you want to remove the "antd" word here?

Comment (fredj) [inline src/pages/3rd-party/index.hbs]: extra `antd`

Comment (marcjansen) [reply src/pages/3rd-party/index.hbs]: The antd word is actually a library…: https://github.com/ant-design/ant-design

Comment (marcjansen) [reply src/pages/3rd-party/index.hbs]: The antd word is actually a library…: https://github.com/ant-design/ant-design

Comment (KaiVolland) [reply src/pages/3rd-party/index.hbs]: Its the name of the ui-package: https://www.npmjs.com/package/antd but maybe "Ant Design" would be better.

Comment (marcjansen) [reply src/pages/3rd-party/index.hbs]: PR welcome @KaiVolland

Comment (elemoine) [reply src/pages/3rd-party/index.hbs]: That was confusing :)

Comment (marcjansen) [reply src/pages/3rd-party/index.hbs]: See #91

**Pull request merged**